src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -571,6 +571,7 @@ public void Handle(ChartUpdatedEvent chartUpdatedEvent)
 
       private void refresh()
       {
+         Chart.SynchronizeDataDisplayUnit();
```

Comment:
Synchronization of data and unit is done upon refresh in the ChartEditorPresenter